cumulusci/tasks/tests/test_metadeploy.py

```diff
@@ -485,6 +494,40 @@ def test_init_task__named_plan(self):
         task._init_task()
         assert expected_plans == task.plan_configs
 
+    @responses.activate
+    def test_init_task_no_plan(self):
+        project_config = create_project_config()
+        project_config.config["project"]["git"]["repo_url"] = "EXISTING_REPO"
+        project_config.keychain.set_service(
+            "metadeploy",
+            "test_alias",
+            ServiceConfig({"url": "https://metadeploy", "token": "TOKEN"}),
+        )
+        task_config = TaskConfig({"options": {"tag": "release/1.0"}})
+        task = Publish(project_config, task_config)
+        with pytest.raises(
+            error,
+            match="No plan found to publish in project configuration",
+        ):
+            task._init_task()
+
+    @responses.activate
+    def test_init_task_no_plan_name(self):
```

Comment:
This test case could, at your option, be parameterized (using `@pytest.mark.parametrize()`) with the previous one.